.github/minty.yaml

```diff
@@ -1,8 +1,24 @@
 version: 'minty.abcxyz.dev/v2'
+
 rule:
-  if: true
```

Comment:
I'm curious why this changed?

Reply:
The original config was built just for integration testing. 

The latest config covers integ and the create-tag workflow as it was defined in the `abcxyz-services` repo.